uint64 framenumber
time reconstruction_stamp
time acquire_stamp
flydra_object[] objects
